ui/src/circle-build.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
```

Comment:
why are these 2 sh files in ui/src instead of .circleci ?

Reply:
I threw the nonspecific utilities into .circleci. These are here because X (in this case, the UI) should know how to build itself. So these are, "I know how to build and deploy myself using Circle." In the back of my mind, I'm always thinking, "How do I keep this sensible for when we finally separate these things into their own repos?"

Reply:
ok makes sense